test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -989,10 +989,13 @@ def test_preempt_checkpoint_release_all(self):
                 exit 0
                 """
         pbs_home = self.server.pbs_conf['PBS_HOME']
```

Comment:
shouldn't this be self.mompbs_conf['PBS_HOME'] since you are making a script on mom?

Reply:
we are just checking pbs home directory.

Reply:
No, you are passing this PBS_HOME read from server's pbs.conf to create a file on mom.  Ideally you should be using PBS_HOME read from mom's pbs.conf file.